### DISCUSSION TITLE
About few mods

### DISCUSSION THREAD

**Discussion by doemty** [UltimatChamp/optimum-fuse#11] [Q&A] [answered]
I wanna ask about 3 mods:
C2ME - Why is it unavaible?
Noisium - Maybe it will be useful? 
Starlight - Same as Noisium
 I ask about these mods because of ReTerraForged, author recommended them so I was interested in them

**Comment by UltimatChamp** [comment, marked as answer]
**C<sup>2</sup>ME:** Already in the Fabric version.  You are probably talking about the (Neo)Forge version. The mod may cause many incompatibilities with Forge mods, as it is natively a Fabric mod. Causes crashes with the default config. The mod needs `threadedWorldGen` and `noTickViewDistance` to be disabled, when trying to run it on Forge.

**Noisium:** Already in the Fabric and (Neo)Forge versions.

**Starlight:** Only useful on the server-side. The client-side usage of the mod is made obselete by the optimizations made by Mojang in 1.20+

**Reply by doemty** [reply]
In 1.9.15 Forge there isn't noisium, I have recently installed it.
About Starlight: but what if I am a host of local server? Does it make sense?
Thanks for reply anyway!<3

**Reply by UltimatChamp** [reply]
> In 1.9.15 Forge there isn't noisium, I have recently installed it.

I'll have to look into that, lol.

> About Starlight: but what if I am a host of local server? Does it make sense?

Yes! _(don't think it will be too noticeable tho)_

> Thanks for reply anyway!<3

<3

**Comment by doemty** [comment]
> **Noisium:** Already in the Fabric and (Neo)Forge versions.
